next.config.js

```diff
@@ -2,7 +2,7 @@ const { withSentryConfig } = require('@sentry/nextjs');
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  output: 'standalone',
+  // output: 'standalone',
```

Comment:
Can you tell me the reason to turn off the `standalone` output when using the custom server?